beacon-chain/core/epoch/precompute/new.go

```diff
@@ -27,7 +27,7 @@ func New(ctx context.Context, state *stateTrie.BeaconState) ([]*Validator, *Bala
 
 	if err := state.ReadFromEveryValidator(func(idx int, val *stateTrie.ReadOnlyValidator) error {
 		// Was validator withdrawable or slashed
-		withdrawable := currentEpoch >= val.WithdrawableEpoch()
+		withdrawable := prevEpoch+1 >= val.WithdrawableEpoch()
```

Comment:
why the change ?

Reply:
To align with the spec. It shouldn't matter since we don't process rewards and penalties until second epoch. See more discussion: https://github.com/ethereum/eth2.0-specs/issues/1846

Either works but I decided to change it to `prevEpoch+1` so it reads exactly the same as the spec, the new comers wont be as confused when they read it